### PR TITLE
updating karma version

### PR DESCRIPTION
**Pull request by afshin**
Fixes breaking `npm run test` command

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "expect.js": "^0.3.1",
     "fs-extra": "^0.26.3",
     "istanbul-instrumenter-loader": "^0.1.3",
-    "karma": "0.13.15",
```
Comment (blink1073) [inline package.json]: Switch back to `^`?

Comment (afshin) [inline package.json]: I'm not sure. This was just a bug, but the same thing could happen in reverse, too, with a newly introduced bug.

Comment (blink1073) [inline package.json]: Hold on a tick, let me check something.

File: package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
...
+    "karma": "0.13.19",
```
Comment (sccolbert) [inline package.json]: what about the `^`?

**Comment by sccolbert** [comment]
Fixed in #13

**Pull request closed without merging**